students/WesleyM/Lesson3/SlicingLab.py

```diff
@@ -0,0 +1,49 @@
+def floor(n):
+    """Rounds down a number if there is a remainder"""
+    res = int(n)
+    return res if res == n or n >= 0 else res-1
+    
+def exchange_first_last(seq):
+    """Reverses the first and last letter or number in a tuple or string"""
+    seqfirst = seq[0:0]
+    seqlast = seq[-1:]
+    remainder = seq[1:-1]
+    a_new_sequence = seqlast + remainder + seqfirst
+    return a_new_sequence
```

Comment:
there is a bug in `seqfirst`, it isn't actually returning the first element.

this simply could be
```
return seq[-1:] + seq[1:-1] + seq[:1]
```